README.md

```diff
@@ -0,0 +1,45 @@
+# Mile High Gophers Job Board #
+
+In order to better share Go-related job openings in the Denver area, and to learn some new things, we are creating a Mile High Gophers Job Board.
+
+Visit the [wiki](https://github.com/WWG-Denver-Boulder/job_board/wiki) for an overview of the project.
+
+Each week we will break up into teams and work on issues related to the PR. Contibutors can take as large or small a role as you are able to.
```

Comment:
I think this should read: "Each week we will break up into teams and work on issues related to the _project_"